bundles/framework/admin-users/AdminRoles.js

```diff
@@ -10,7 +10,7 @@ Oskari.clazz.define(
         this.instance = parent;
         this._localization = localization;
         this.sandbox = parent.getSandbox();
-        this.ajaxUrl = this.sandbox.getAjaxUrl();
+        this.ajaxUrl = Oskari.urls.getRoute();
```

Comment:
Looks like you can add 'ManageRoles' as param and modify the usage by removing action_route=ManageRoles.